Summary:
Cluster 0 :
 For the MVA prME vaccinated group, all mice reduced viral loads in the blood: reduced viral load peak was detected in three out of five mice, and the other two mice presented a vir
al load peak at day 2, instead of day 3.
 Besides that, the protection efficacy of MVA Env \u2206TM was better than the rest of the MVA-ZIKV vaccines, by substantially reducing the ZIKV loads in blood, such protection did 
not reach 100% efficacy and we did not detect statistical significance between the MVA-ZIKV vaccinated groups when analysing three-day post challenge viral loads.
 The estimate\u03b1 is obtained from the development of the number of infectious people over time between the time the 100th individual becomes infectious and the time the 1000th in
dividual becomes infectious, by using least square estimation of the natural logarithm of the number of infecteds against time.
 The estimate of R 0 based on the real infection tree is obtained by looking at the epidemic from a generation perspective: All individuals infected by the initially infectious indi
viduals are in generation 1, individuals infected by generation 1 infectives are in generation 2 etc.
 Other metabolites identified, although at lower levels (<10% of parent [Pyr 1 ]apelin-13) included [Pyr 1 ]apelin-13 (1) (2) (3) (4) (5) (6) (7) (8) , [Pyr 1 ]apelin-13 (1) (2) (3)
 (4) (5) (6) (7) and [Pyr 1 ]apelin-13 (1) (2) (3) (4) (5) that are likely to be biologically inactive.
 We have recently published on an another approach to stabilise apelin peptides in human blood using albumin domain (AlbudAb) antibody conjugated to an apelin analogue, MM202 and sh
owed that this peptide was resistant to degradation yet retained biological activity at the human apelin receptor in vitro and in vivo 9 .
 For the first time in humans in vivo we have identified as the most abundant metabolite of [Pyr 1 ]apelin-13, the ACE2 cleavage product, [Pyr 1 ] apelin-13 (1) (2) (3) (4) (5) (6) 
(7) (8) (9) (10) (11) (12) that we have previously demonstrated retains significant biological activity in addition to the putative neprilysin metabolites [Pyr 1 ]apelin-13 (4) (5) 
(6) (7) (8) (9) (10) (11) (12) (13) and [Pyr 1 ]apelin-13 (6) (7) (8) (9) (10) (11) (12) (13) .
 We also thank the Director Boards and all the doctors, nurses and technicians of the infectious diseases departments, emergency departments, and clinical laboratories of the Quang 
Nam Central General Hospital, Quang Nam Northern Mountainous Region General Hospital, Quang Nam Regional General Hospital and Quang Nam Provincial General Hospital, Quang Nam provin
ce, Vietnam, for their support with the sample collection.
 The DENV-encoded proteins NS2A, NS4A and NS4B, expressed separately in human alveolar basal epithelial cells (A549), enhanced replication of IFN-\u03b1/\u03b2-sensitive viruses in 
the presence of IFN-\u03b1/\u03b2, and NS4B strongly inhibited IFN-\u03b1/\u03b2 stimulation of ISRE promoter [59] .
 For example, policies should concentrate on educating people working in urban areas to take care in their sexual relations while in the urban area by promoting safe sex, encouragin
g condom use, reducing the number of sex partners or reducing risky behaviours like having sex with sex workers [12, 15, 61] .

Cluster 1:

Previously, several studies were conducted on synonymous codon usage bias in a wide variety of organisms including prokaryotes and eukaryotes [10] [11] [12] [13] [14] [15] [16] , and till date in many organisms the codon usage patterns have been interpreted for diverse reasons.
 In unicellular and multicellular organisms it was observed that, preferred synonymous codons/optimal codons with abundant tRNA gene copy number rise with gene expression level within the genome that supports selection on high codon bias confirmed by positive correlation between optimal codons and tRNA abundance [18, 22, 27] .
 However, Comeron (2004) observed that in human genome, highly expressed genes have preference towards codon bias favoring codons with most abundant tRNA gene copy number compared to less highly expressed genes [28].
 3) depicted that the codon GCC, CGC (except Rattus norvegicus), ATC (except Tupaia chinensis), CTG, ACC (except Rattus norvegicus, Macaca mulatta), GTG (except Felis catus) were displayed as the over represented codons (RSCU>1.
 According to Yang and Nielsen (2008) , codon bias in mammals is mainly influenced by mutation bias and the selection on codon bias is weak for nearly neutral synonymous mutations [40] .
 Epidemiological investigations identified secondary transmission in 24 index patients, including 4 which were involved in super-spreading events; 10 of the 24 index patients caused secondary transmission within TTSH itself, while another 14 index patients caused secondary transmission in other healthcare institutions and/or the community.
 There were also indications that index patients with more severe illness were more likely to transmit the disease; while 4/ 22 (18%) of the index patients who transmitted the disease had been admitted to ICU or HD by the time of isolation, this was observed in none of the index patients who did not transmit the disease (p = 0.
 Furthermore, a range of potential mechanisms may not be adequately represented by this predictor set; a lack of an effect of rainfall, for example, does not discount the potential for other climatic factors to play a role, and a lack of an effect of latitude could mean that it is simply a poor proxy for other more meaningful factors that nevertheless exhibit some latitudinal variation (e.
 BRTs do not provide p-values or coefficients, but rank variables by their relative influence in explaining variation in the outcome 26 The current study aims to better analyze the mechanistic underpinnings of disease emergence for zoonotic EIDs of wildlife origin, while addressing some methodological limitations of Jones et al.
 We selected a refined set of spatial predictors for their relevance to a priori hypotheses on plausible mechanisms underlying zoonotic EID emergence, including proxies for human activity, environmental factors, and the zoonotic pathogen pool from which novel diseases could emerge, all key features of conceptual models of zoonotic spillover [7] [8] [9] [10] [11] .

Cluster 2:

For the last two decades, monoclonal antibodies and antibody fragments have been proven to be effective as therapeutic and diagnostic agents, and have long been invaluable tools in various fields of biological research [1, 2] . 
For the development of antigenspecific antibodies, hybridoma technology that relies on animal immunization has been traditionally employed [3] . 
The recent progress in combinatorial technologies because of in vitro antibody repertoires and high-throughput screening methodologies has allowed the development of target-specific antibodies without animal immunization [4, 5] .
In these technologies, various protein display systems including phage display, ribosome display, and cell-surface display, have been widely used for the initial isolation of antibodies specific to antigens from huge libraries, as well as for engineering the antibodies towards desired functions, e.g., enhanced affinity and higher thermostability. 
However, the most recent tools require repeated screenings in order to isolate potential candidates from the library, and consequently, they require relatively long time periods (several days to weeks) to complete the screening. 
The recent emergence and rapid dissemination of new viruses that cause serious human and animal diseases, such as SARS coronavirus, swine flu H1N1 virus, and avian influenza H5N1 virus, has raised world concerns. 
The following strategy is usually used for screening a recombinant antibody library: (i) cultivation of library cells; (ii) fluorescent-antigenpeptide or protein labeling of the library cells; (iii) FACS sorting of the highly fluorescent population; (iv) regeneration of the sorted cells by regrowth or re-cloning of the sorted target genes; (v) repetition of steps i-iv until a highly fluorescent population is separated from the negative control population; and (vi) analysis of the individual clones.
Among these steps, the step determining the screening time is the regeneration of the sorted cells (step iv). 
In all of the current screening strategies, the sorted cells need to be regenerated for the next round of sorting, which can be done by cultivating the cells for at least one day orby re-cloning the genes, which takes several days .
In addition to the regeneration time, contamination of the sorted cells by nonspecific clones also needs to be considered. During the cultivation for regeneration of sorted cells, differential growth rates among various clones (particularly non-specific clones) due to unregulated protein expression and differing cell viability can decrease the library screening efficiency, resulting in more rounds of sorting (longer duration) to isolate the potential antibody candidate [12] .
Herein, we report the development of a new high-throughput screening strategy based on Escherichia coli protein display and FACS sorting, which allows the simple and rapid isolationof potential candidates from a huge library in one day.
Finally, we constructed the fully synthetic human antibody library in which antibody fragments (single-chain variable fragment, scFv) were produced in the periplasm of E. coli. 

Cluster 3 :
                          
The whole FACS screening rounds of the synthetic human antibody library against each viral antigen could be done in one day, and these results show that repeated FACS screening without regeneration of the sorted cells can be a rapid and efficient method to isolate potential antibody candidates in case of urgent requirements. 
The bacterial strains and plasmids used in this study are listed in the Table S1 . E. coli Jude-1 was used as the main host for gene cloning and library screening. 
E. coli HM130 was used for the production and purification of the isolated antibodies (scFv). 
The E. coli cells were inoculated into Luria-Bertani (LB) medium (10 g/L of tryptone, 5 g/L of yeast extract, and 10 g/L of NaCl) containing 2% (w/v) glucose. 
After an overnight cultivation at 37uC and 200 rpm, the cells were transferred into 100 mL of fresh LB medium without glucose in a 500-mL flask and incubated at 37uC and 200 rpm.
When the cell density (OD 600 ) reached 0.6, the cells were induced with 1 mM isopropyl-b-D-thiogalactopyranoside (IPTG) and were further cultivated at 25uC at 200 rpm for 4 h.
The cells were harvested by centrifugation at 6,000 rpm for 10 min at 4uC for further analysis. 
In all the cultivations, ampicillin (50 mg/mL) was used as the sole antibiotic.\nFor the FACS screening of a human synthetic antibody (scFv) library, three fluorescent antigen probes were chemically synthesized: (i) FITC-CRDNWHGSNRPW as an N1 epitope of H1N1 influenza virus [13] ; 
One aliquot (100 mL) of the library stocks frozen at 280uC was thawed and inoculated into 100 mL of LB medium containing 2% (w/v) glucose and ampicillin.
After cultivation under the condition described above, the cells were harvested by centrifuging for 10 min at 6,000 rpm and 4uC. 
For efficient labeling with fluorescent probes, cell pellets were resuspended in 56 Tris-KCl buffer (150 mM Tris-HCl at pH 7.4 and 750 mM KCl), which dramatically increased the permeability of the E. coli outer membrane and allowed the fluorescent antigen probes to permeate into the periplasm [9, 16] . 
The resuspended cells were incubated with 5 mM of antigen peptides (N1, PreS2, or VP1 epitope) conjugated with FITC for 1 h at 4uC. 
The cells were then washed twice with the same buffer (56 Tris-KCl), and the fluorescent probe-labeled cells were sorted using a high-speed flow cytometer (Moflo XDP, Beckman Coulter, Miami, FL). 
In FACS sorting, the cells were selected on the basis of high fluorescence intensity detection through a 530/40 band-pass filter for obtaining the FITC emission spectrum. 
Purify mode'' was used as the sorting mode, which sorts only those drops that contain positive cells.  
coli cells sorted in each round of screening were immediately reused for the next round of FACS sorting without regeneration. 
The sorting was repeated until the highly fluorescent population was fully enriched. 

Cluster 4 :
After the final sorting, the scFv genes were amplified from the sorted cells by performing PCR with primers, Assembly-F and Assembly-R.
After digestion with SfiI, the amplified scFv genes were cloned into pMoPac16 and transformed into E. coli Jude-1 for the further analysis of the single clones for enrichment of cells producing FlAsH tag (FLNCCPGCCMEP) from non-specific cells, cells harboring pMoPac16-MBP were diluted with non-specific cells harboring pMoPac16 at 1:10000 ratio. 
After labeling with with 25 mM of FlAsH-EDT 2 (Invitrogen), the mixed cells were applied to FACS and the fluorescent cells were sorted by the repeated FACS screening as described above for the production of the isolated antibody fragments.
After cultivation in 500 mL of LB medium, the cells were harvested by centrifugation for 10 min at 6,000 rpm and 4uC. 
The cells were  with PBS and resuspended in the same buffer of Crude extracts of the cells were prepared by sonication (20 min at 50% pulse and 20% amplitude), and the extracts were centrifuged for 10 min at 10,000 rpm and 4uC to yield soluble lysates. 
The soluble lysates were filtered through a 0.45-mm syringe filter, and the soluble lysates were poured into Poly-prep chromatography columns (Bio-Rad, Hercules, CA) filled with Talon metal-affinity resin (Clontech, Mountain View, CA). 
The purified scFvs were used for ELISA analysis for the preparation of GST-fused antigens (N1 of H1N1, PreS2 of HBV, and VP1 of FMDV), three vectors were constructed with the primers used in the construction of the GST-fused antigens are listed in the Table S2 . 
The plasmid pGEX-4T-1 containing the GST gene was used as the template, and the N1 epitope sequence (CRDNWHGSNRPW) of the H1N1 influenza virus [13] was fused to the C-terminus of GST by performing PCR with the primer sets GST-F and GSTN1-R. 
For the synthesis of the PreS2 epitope sequence (NSTTFHQALLDPRVRGLYFPAGG) of HBV [14] and the VP1 epitope sequence (PVTNVRGDLQVLAQK) of FMDV [15] , the same PCR strategy was used, except that the reverse primers-GSTPreS2-R and GSTVP1-R-were used for synthesizing the PreS2 and VP1 epitope genes, respectively. 
E. coli Jude-1 harboring pMoPac1-GST-N1, pMoPac1-GST-PreS2, or pMoPac1-GST-VP1 were cultivated in 500 mL of LB medium in a 2-L shaking flask at 37uC at 200 rpm. 
When the cell density (OD 600 ) reached 0.6, the cells were induced with 1 mM of IPTG and were further cultivated at 37uC at 200 rpm for 6 h and the cells were harvested by centrifugation at 6,000 rpm for 10 min at 4uC. 
The cells were then resuspended in 50 mL of PBS and were lysed by sonication (20 min at 50% pulse and 20% amplitude) after filtration with a 0.45-mm syringe filter 
The GST-fused antigens were purified by using Glutathione Sepharose resin (GE Healthcare Biosciences AB, Uppsala, Sweden) as per the method specified by the manufacturer.\nThe GST-fused antigens were mixed with 0.05 M carbonatebicarbonate coating buffer (pH 9.6) to a final concentration of 2 mM. The antigen solution (100 mL) was loaded onto 96-well ELISA plate, which was then incubated for 2 h at 37uC.

Cluster 5 :  
The fluorescence of the sorted cells was clearly observed to be fully separated from that of the original Black bar indicate signals from BSA coated wells, and grey bars indicate signals from GST-fused N1 antigen coated wells.
ELISA analysis with the purified scFvs of Symbols: circle, square and triangle represent the purified antibody from clones S1, S5 and S16, respectively of the closed and open symbols represent the coating of GST-fused N1 antigen and BSA on 96-well plates, respectively.  
Finally, 1,561 fluorescent cells were collected and used for analysis of individual clones and the sorting results are summarized in the Table 1 after the final screening. 
The scFv genes of the 1,561 sorted cells were rescued by PCR, and the PCR product was cloned into pMoPac16 for the periplasmic expression of scFvs after transformation into E. coli Jude-1, a total of 16 colonies were randomly picked from the agar plate, and the binding activities of the isolated antibodies were analyzed by performing ELISA. 
Most clones, except for one (S10), exhibited higher binding activity against the N1 peptide than the non-specific BSA control (Fig. 3B) of the sequencing experiments, we found that among the 16 clones, six clones including S2, S4, S6, S9, S11, and S13, had the same DNA sequences.  
The same clones with the same sequences showed similar levels of binding activity, except for S3 and S10.
The scFv genes were transformed into the E. coli strain HM130, which is protease-deficient and suitable for higher scFv production [21] after cultivation, scFvs of three clones S1, S5, and S7 could be purified with high purity, but we failed to purify scFv of clone S16 (data not shown). 
The specific activities of  against the N1 epitope were analyzed by ELISA and we observed that the binding activities of S1 and S5 were higher than that of S7 and a non-specific BSAcontrol of the sequence information of three isolated scFv (S1, S5 and S7) was provided.
To demonstrate the general use of our strategy for antibody screening, we also isolated a potential antibody against another target, PreS2 of HBV with after cultivation and permeabilization of the library, cells were labeled with PreS2 antigenic peptide conjugated with FITC, and the labeled library cells were sorted by FACS. r population, and finally 1,412 cells were collected

